### ISSUE TITLE
sndev start :  .env.development: not found

### ISSUE THREAD

**Issue by riccardobl** [stackernews/stacker.news#1297]
```
./sndev start
```
exists with 
`./sndev: 5: .: .env.development: not found`
in the current master branch.
It used to work fine before 8000886e72ca5c0c7dcdbd1fdf7892765c7776b1

Replacing line 5 
https://github.com/stackernews/stacker.news/blob/53465e3f4623dff71d0f13a5f2e8da9286407d3e/sndev#L5
with
`. $PWD/.env.development`
makes it work again

**Comment by huumn** [comment]
Oh weird. Maybe that's an os-specific thing. On mac, it's implicitly a path relative to the working directory.

**Comment by riccardobl** [comment]
Could be, i'm on Ubuntu 24.04 LTS

**Comment by huumn** [comment]
I pushed a fix to master using`./.env.development`. lmk if that's still broken

**Comment by ekzyis** [comment]
That works for me without symlinks now.

Before, it was trying to find .env.development in the same folder the script was in but since I symlinked `sndev` to ~/.local/bin which is in my PATH it searched for it in ~/.local/bin and thus didn't find it. To fix this, I also symlinked .env.development to ~/.local/bin.

Now it looks it up in the folder I am running the command.

Maybe @riccardobl also used a symlink for the script?
